homeassistant/components/home_connect/light.py

```diff
@@ -93,30 +113,6 @@ def __init__(self, device, desc, ambient):
             self._color_key = BSH_AMBIENT_LIGHT_COLOR
             self._attr_color_mode = ColorMode.HS
             self._attr_supported_color_modes = {ColorMode.HS}
-        elif desc == "CoolingLight":
```

Comment:
Realized this needed to be check before the Entity was instantiated for appliances that may have more than one type of light i.e. a refrigerator with both Internal and External lights.